From a4a831f998d513c6d01394d9b106aa42faf60dd3 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Sat, 23 Feb 2019 21:05:28 +0200
Subject: [PATCH] build: always use internal synctex instead of automagically

We don't have a separate synctex package and would have to pull in
all of texlive-core; so instead of fixing the automagic use of it,
just remove support for external synctex, as that's easier.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 13bdd600..e5334319 100644
--- a/configure.ac
+++ b/configure.ac
@@ -539,7 +539,7 @@ GOBJECT_INTROSPECTION_CHECK([1.0])
 
 dnl ================== libsynctex ===========================================
 
-PKG_CHECK_MODULES(SYNCTEX, [synctex >= $SYNCTEX_REQUIRED], has_synctex=system, has_synctex=internal)
+has_synctex=internal
 
 dnl not found? use internal code copy.
 if test "x$has_synctex" = "xinternal"; then
-- 
2.17.0

